src/sentry/utils/math.py

```diff
@@ -30,3 +30,33 @@ def mad(values, K=1.4826):
     # http://en.wikipedia.org/wiki/Median_absolute_deviation
     med = median(values)
     return K * median([abs(val - med) for val in values])
+
+
+def nice_int(x):
```

Comment:
`nice_int` is implicitly tested by the histogram function, but should we add some explicit tests for it too?